app/assets/stylesheets/rails_admin.scss

```diff
@@ -0,0 +1,2 @@
+$fa-font-path: ".";
```

Comment:
Basically because we don't know specifically where the font files are kept. We know that through the magic of RailsAdmin, the font are are made available. (But also, this file is auto-generated when you install RailsAdmin, so I just trust it to do the right thing)